src/locales/pl.json

```diff
@@ -0,0 +1,171 @@
+{
+  "__MyLocaleLanguage__": "Polski",
+  "__maintaner__": "https://github.com/aPoCoMiLogin",
+
+  "title": "YakYak - Klient Hangouts",
+  "description": "Aplikacja desktopowa Google Hangouts",
+
+  "settings": "Ustawienia",
+  "recent": "Ostatnie",
+  "details": "Szczegóły",
+
+  "connection" :{
+    "connecting": "Łączenie",
+    "loading": "Ładowanie kontaktów",
+    "not_connected": "Brak połączenia (sprawdź swoje połączenie)"
+  },
+
+  "input": {
+    "message": "Wiadomość",
+    "emoticons": "Pokaż emotikony",
+    "image": "Dołącz zdjęcie",
+    "typing": "pisze"
+  },
+  "conversation": {
+    "title": {
+      "one": "Rozmowa",
+      "other": "Rozmowy"
+    },
+    "numbered": "Rozmowa %d",
+    "options": "Opcje rozmowy",
+    "setting": "Ustawienia rozmowy",
+    "add": "Dodaj nową rozmowę",
+    "new": "Nowa rozmowa",
+    "name": "Nazwa rozmowy",
+    "edit": "Edytuj rozmowę",
+    "multiuser": "Utwórz grupową rozmowę",
+    "delete": "Usuń rozmowę",
+    "delete_confirm": "Na pewno chcesz usunąć rozmowę?",
+    "leave": "Opuść rozmowę",
+    "leave_confirm": "Na pewno chcesz opuścić rozmowę?",
+    "search": "Szukaj ludzi",
+    "new_message": "Nowa wiadomość"
+  },
+  "notification": {
+    "one": "Powiadomienie",
+    "other": "Powiadomienia"
+  },
+  "favorite": {
+    "title": {
+      "one": "Ulubiony",
+      "other": "Ulubione"
+    },
+    "star_it": "Dodaj/Usuń z ulubionych"
+  },
+  "actions": {
+    "ok": "OK",
+    "cancel": "Anuluj"
+  },
+  "call": {
+    "incoming": "Połączenie przychodzące",
+    "incoming_from": "Połączenie przychodzące od %s",
+    "accepted": "Odebrane",
+    "accept": "Odbierz",
+    "rejected": "Odrzucone",
+    "reject": "Odrzuć",
+    "missed": "Nieodebrane połączenie od %s"
+  },
+  "menu": {
+    "title": "Menu",
+    "file": {
+      "title": "YakYak",
+      "hide": "Ukryj YakYak",
+      "hide_others": "Ukryj pozostałe",
+      "show": "Pokaż wszystko",
+      "inspector": "Otwórz inspektor",
+      "logout": "Wyloguj",
+      "quit": "Zamknij"
+    },
+    "edit": {
+      "title": "Edycja",
+      "undo": "Cofnij",
+      "redo": "Ponów",
+      "paste": "Wklej",
+      "cut": "Wytnij",
+      "copy": "Kopiuj",
+      "copy_link": "Kopiuj adres linku",
+      "copy_image": "Kopiuj grafikę",
+      "select_all": "Zaznacz wszystko",
+      "language": "Język"
```

Comment:
can you change to: I'll have to change the template as this is not obvious and I've been doing this personally

"language": "Język (Language)"